CHANGELOG.md

```diff
@@ -1,3 +1,26 @@
+## 0.30.0 (2017-11-07)
+
+#### BREAKING CHANGES
+
+* Assign public ports in development/production blocks of `service.yml`. `docker` block has been removed. Only a single port is supported at this time.
+```
+docker:
+  ports:
+    - '3000:3000'
+```
```

Comment:
should also mention `production.public-port` that was renamed tp `production.port`